skymp5-client/src/features/loadOrder.ts

```diff
@@ -143,13 +147,13 @@ export const verifyLoadOrder = () => {
           'LOAD ORDER ERROR!\nHowever, ignoring it because of ignoreLoadOrderMismatch being set.' +
           '\nExpect EVERYTHING BREAK, unless you know what you are doing.\nCheck console for details.' +
           '\nThis message will disappear after 30 seconds.',
-          [255, 0, 0, 1], 30,
+          [255, 0, 0, 1], getDefaultFont(), 30,
         );
         return;
       }
       updateText(
         'LOAD ORDER ERROR!\nCheck console for details.',
-        [255, 0, 0, 1],
+        [255, 0, 0, 1], "Data/Platform/Fonts/Tavern.spritefont",
```

Comment:
Probably `getDefaultFont()` here too. But there's actually no need in function here btw. Maybe just `const defaultFont = '...';`